lib/shopify_transporter/exporters/magento.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require_relative 'magento/magento_exporter.rb'
```

Comment:
It's probably ok to call this file/class `exporter`/`Exporter`, given that it's already in a `magento` directory.

Reply:
I was just a bit worried about ambiguous naming potentially leading to future issues, similar to the `RACC::ParseError` stuff we encountered in the App. Do we think that having identically named classes in nested modules is okay?